discord_bot/src/commands/verify.ts

```diff
@@ -0,0 +1,59 @@
+import { ChatInputCommandInteraction, InteractionResponse, SlashCommandBuilder } from "discord.js";
+import { Command } from "./index";
+import { PrismaClient } from '@prisma/client';
+import { checkin } from '../config';
+
+
+
+const prisma = new PrismaClient();
+
+
+export const verifyCommand = {
+  data: new SlashCommandBuilder()
+    .setName("verify-new")
+    .setDescription("Verify Users For MakeUC!")
+    .addStringOption(option =>
+      option.setName('email')
+        .setDescription('Insert the email address the hacker used to register for MakeUC')
+        .setRequired(true)),
+  execute: async (interaction: ChatInputCommandInteraction) => {
+    const email = interaction.options.getString('email');
+    console.log(email);
+    let reply = "";
+
+    if (checkin === "close") {
```

Comment:
Can we change this to be `!== "open"`? Then, anything that isn't open will not count.

Reply:
We should also probably early return